Set rv->compressed for instruction ecall and ebreak

qwe661234 opened the pull request:
The exception handler is invoked by the instructions ecall and ebreak, thus we must set rv->compressed for these instructions.

jserv reviewed (changes requested):
Again, check your git commit message.

qwe661234 committed:
Set rv->compressed for instruction ecall and ebreak

The exception handler is invoked by the instructions ecall and ebreak,
thus we must set rv->compressed for these instructions.
